scarb/src/compiler/plugin/proc_macro/ffi.rs

```diff
@@ -147,8 +147,8 @@ impl ProcMacroInstance {
         // Free the memory allocated by the `stable_token_stream`.
         // This will call `CString::from_raw` under the hood, to take ownership.
         unsafe {
-            TokenStream::from_owned_stable(stable_result.input);
-            TokenStream::from_owned_stable(stable_result.input_attr);
+            TokenStream::free_owned_stable(stable_result.input);
```

Comment:
Change comment in [this](https://github.com/software-mansion/scarb/blob/308f593261681a0c03a718ae3f985257fab34c67/scarb/src/compiler/plugin/proc_macro/ffi.rs#L136C53-L136C70) line, from `from_owned_stable` to `free_owned_stable`